src/redux/categories/categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>Categories Section</div>
+);
+
```

Comment:
- [ ] Same for the Category component kindly place it in the components folder.

- [ ] In addition, the category component should receive the category as a prop.